### ISSUE TITLE
html.report: step.text contains HTML elements in rendered HTML page

### ISSUE THREAD

**Issue by jenisys** [hrcorval/behavex#79]
**Describe the bug**
If a scenario contains steps with `step.text` (multi-line docstring), the output of this scenario contains `<br/>` HTML elements that are not part of the original output. 

* MAYBE: `<br/>` is contained in a `<pre> ... </pre>` HTML element ?!?

EXAMPLE:

```gherkin
# -- HINT: Copied from HTML page in web browser
Scenario:  Duplicated Step in same File   <-- COLLAPSABLE SECTION: For Scenario is here
  Given a new working directory
  And a file named "features/steps/alice_steps.py" with
    from behave import given, when, then<br><br>@given(u'I call Alice')<br> ... <-- PROBLEM-POINT 1 is HERE
  And a file named "features/duplicated_step_alice.feature" with
    Feature:<br> Scenario: Duplicated Step<br> Given I call Alice ...    <-- PROBLEM-POINT 2 is HERE.
```

**To Reproduce**
1. Run `behavex` with [step.duplicated_step.feature](https://github.com/behave/behave/blob/main/features/step.duplicated_step.feature#L8) and inspect the HTML report after the test run

**Expected behavior**

* Rendered HTML should not contain HTML elements (in `step.text` or ...), like: `<br/>`

**Version Info:**
 - behavex v2.0.2 (current HEAD of repository)

**Comment by anibalinn** [comment]
Fix implemented in the upcoming release branch [release_2.0.2](https://github.com/hrcorval/behavex/tree/release_2.0.2)

**Comment by anibalinn** [comment]
Fix released in version 3.0.0: https://pypi.org/project/behavex/3.0.0/
branch [release_3.0.0](https://github.com/hrcorval/behavex/tree/release_3.0.0)
Thanks!
